lib/android.js

```diff
@@ -257,6 +257,7 @@ function _getApi () {
           deoptimize(instrumentation);
         };
       },
+      _ZN3art7Runtime19DeoptimizeBootImageEv: ['art::Runtime::DeoptimizeBootImage', 'void', ['pointer']],
```

Comment:
Are we sure this is available on all ART versions? If not we should mark it as optional.

Reply:
I am checking this now. I have android 7, 10, and 11 devices to check on. Otherwise I can do emulator. But from the AOSP it looks like its there for > android 8.

Reply:
We should at least make sure it's available on 5.0, which IIRC is the oldest version where ART was default. It's also available before behind a flag IIRC, but I'm not sure we ever tested that anyway. (Oldest Android version supported is currently 4.3.)